src/components/Badge/Badge.driver.ts

```diff
@@ -13,10 +13,15 @@ const hasPriority = async (
   return priorityVale === priority;
 };
 
+const hasIcon = async (base: UniDriver): Promise<boolean> => {
+  return !!base.$(`[data-hook=badge-icon] svg`);
+};
+
 export interface BadgeDriver extends BaseUniDriver {
   isPrimary(): Promise<boolean>;
   isLight(): Promise<boolean>;
   isDefault(): Promise<boolean>;
+  getIcon(): Promise<boolean>;
```

Comment:
so I think we should change this to `hasIcon` cause we're not really returning the icon, but a boolean

Reply:
You are right, changed it, I had to change the inner "hasIcon" function because it would shadow it, changed the inner one to "isIconExists"